vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -181,48 +181,48 @@ void Unblinded::MakeContribution(
     ledger::UnblindedTokenList list) {
   const auto reconcile = ledger_->GetReconcileById(viewing_id);
 
-  if (reconcile.type_ == ledger::RewardsType::ONE_TIME_TIP ||
-      reconcile.type_ == ledger::RewardsType::RECURRING_TIP) {
+  if (reconcile.type == ledger::RewardsType::ONE_TIME_TIP ||
+      reconcile.type == ledger::RewardsType::RECURRING_TIP) {
     const auto callback = std::bind(&Unblinded::ContributionCompleted,
         this,
         _1,
         viewing_id);
     SendTokens(
-        reconcile.directions_.front().publisher_key_,
-        reconcile.type_,
+        reconcile.directions.front().publisher_key,
+        reconcile.type,
         std::move(list),
         callback);
     return;
   }
 
-  if (reconcile.type_ == ledger::RewardsType::AUTO_CONTRIBUTE) {
+  if (reconcile.type == ledger::RewardsType::AUTO_CONTRIBUTE) {
     PrepareAutoContribution(viewing_id, std::move(list));
   }
 }
 
 bool Unblinded::GetStatisticalVotingWinner(
     double dart,
-    const braveledger_bat_helper::Directions& directions,
+    const ledger::ReconcileDirections& directions,
     Winners* winners) const {
   if (!winners) {
     return false;
   }
 
   double upper = 0.0;
   for (const auto& item : directions) {
-    upper += item.amount_percent_ / 100.0;
+    upper += item.amount_percent / 100.0;
     if (upper < dart) {
       continue;
     }
 
-    auto iter = winners->find(item.publisher_key_);
+    auto iter = winners->find(item.publisher_key);
 
     uint32_t current_value = 0;
     if (iter != winners->end()) {
-      current_value = winners->at(item.publisher_key_);
-      winners->at(item.publisher_key_) = current_value + 1;
+      current_value = winners->at(item.publisher_key);
+      winners->at(item.publisher_key) = current_value + 1;
     } else {
-      winners->emplace(item.publisher_key_, 1);
+      winners->emplace(item.publisher_key, 1);
     }
 
```

Comment:
The lines from 218-226 can be rewritten with one line:
`(*winners)[item.publisher_key]++;`
Default is  zero-initialized.

Reply:
@gdregalo Unrelated to this PR. Data structure variable was just renamed from `publisher_key_` to `publisher_key`